<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="20517" started-at="2016-05-02T14:14:26Z" finished-at="2016-05-02T14:14:47Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="20517" started-at="2016-05-02T14:14:26Z" finished-at="2016-05-02T14:14:47Z">
      <class name="LoginTestCases.LoginPostive1">
        <test-method status="PASS" signature="launchBrowser()[pri:0, instance:LoginTestCases.LoginPostive1@1089de1]" name="launchBrowser" is-config="true" duration-ms="7577" started-at="2016-05-02T19:44:26Z" finished-at="2016-05-02T19:44:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchBrowser -->
        <test-method status="PASS" signature="passCollegeApplicationUrl()[pri:0, instance:LoginTestCases.LoginPostive1@1089de1]" name="passCollegeApplicationUrl" is-config="true" duration-ms="2884" started-at="2016-05-02T19:44:34Z" finished-at="2016-05-02T19:44:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- passCollegeApplicationUrl -->
        <test-method status="FAIL" signature="loginMethod()[pri:0, instance:LoginTestCases.LoginPostive1@1089de1]" name="loginMethod" duration-ms="10036" started-at="2016-05-02T19:44:37Z" finished-at="2016-05-02T19:44:47Z">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[javax.mail.MessagingException: IOException while sending message;
  nested exception is:
	java.io.FileNotFoundException: D:\kuldeep.jpg (The system cannot find the file specified)]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: javax.mail.MessagingException: IOException while sending message;
  nested exception is:
	java.io.FileNotFoundException: D:\kuldeep.jpg (The system cannot find the file specified)
	at LoginTestCases.LoginPostive1.sendAttachmentThroughEmail(LoginPostive1.java:278)
	at LoginTestCases.LoginPostive1.takesScreenshot(LoginPostive1.java:188)
	at LoginTestCases.LoginPostive1.loginMethod(LoginPostive1.java:165)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
Caused by: javax.mail.MessagingException: IOException while sending message;
  nested exception is:
	java.io.FileNotFoundException: D:\kuldeep.jpg (The system cannot find the file specified)
	at com.sun.mail.smtp.SMTPTransport.sendMessage(SMTPTransport.java:1245)
	at javax.mail.Transport.send0(Transport.java:254)
	at javax.mail.Transport.send(Transport.java:124)
	at LoginTestCases.LoginPostive1.sendAttachmentThroughEmail(LoginPostive1.java:273)
	... 26 more
Caused by: java.io.FileNotFoundException: D:\kuldeep.jpg (The system cannot find the file specified)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.<init>(FileInputStream.java:146)
	at javax.activation.FileDataSource.getInputStream(FileDataSource.java:97)
	at javax.activation.DataHandler.writeTo(DataHandler.java:305)
	at javax.mail.internet.MimeBodyPart.writeTo(MimeBodyPart.java:1627)
	at javax.mail.internet.MimeBodyPart.writeTo(MimeBodyPart.java:961)
	at javax.mail.internet.MimeMultipart.writeTo(MimeMultipart.java:553)
	at com.sun.mail.handlers.multipart_mixed.writeTo(multipart_mixed.java:103)
	at javax.activation.ObjectDataContentHandler.writeTo(DataHandler.java:884)
	at javax.activation.DataHandler.writeTo(DataHandler.java:317)
	at javax.mail.internet.MimeBodyPart.writeTo(MimeBodyPart.java:1627)
	at javax.mail.internet.MimeMessage.writeTo(MimeMessage.java:1849)
	at com.sun.mail.smtp.SMTPTransport.sendMessage(SMTPTransport.java:1196)
	... 29 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginMethod -->
        <test-method status="PASS" signature="closeBrowser()[pri:0, instance:LoginTestCases.LoginPostive1@1089de1]" name="closeBrowser" is-config="true" duration-ms="0" started-at="2016-05-02T19:44:47Z" finished-at="2016-05-02T19:44:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- LoginTestCases.LoginPostive1 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
